### PR TITLE
Updated elysia-session so it works as expected

### PR DESCRIPTION
**Pull request by jkelkar**
Hi,
I needed this plugin but never got it to work with saving data. I am new to elysia and to the best of my knowledge I have updated your code to enable saving and accessing data saved in session.
Update your repo if you find it useful. I am using the changed version in projects and it seems to work well.

Regards,
Jay Kelkar
P.S. Thanks for getting this started.

### CODE REVIEW

**Comment by ultimateshadsform** [comment]
I don't think this is going to be updated anymore. It's probably better to just upload to a new repo and just create a new npm package since original repo owner doesn't update or accept PRs.

**Comment by jkelkar** [comment]
Looks like you may be right. I will release a new repo when I can, currently it is not a priority.

Jay

File: package.json [hunk 1]
```diff
@@ -79,6 +79,7 @@
   },
   "dependencies": {
     "iron-webcrypto": "^0.10.1",
-    "nanoid": "^5.0.1"
+    "nanoid": "^5.0.1",
+    "typescript": "^5.4.5"
```
Comment (gaurishhs) [inline package.json:83]: Can typescript be moved to devDependencies?

**Comment by gaurishhs** [comment]
hi, i haven't opened github for a while (exams), I don't think so i ever got an email for this PR. i will merge your PR asap.

File: package.json (hunk 1 above)
Comment (ultimateshadsform) [reply package.json:83]: Yeah that's where it should be

**Comment by jkelkar** [comment]
Yes, typescript can be moved to devDependencies. By the way, the solution I have proposed is ok, but not perfect. I have abandoned this approach and am using fastify to finish my project, it has a reliable session package. 

I also found elysia a bit hard to work with in its present state for use in a complex server.

**Comment by gaurishhs** [comment]
No issues. I'll merge this PR and make the necessary changes

**Comment by jkelkar** [comment]
Thanks.

Regards,
Jay

**Pull request merged**